cdk/new_relic/transform.js

```diff
@@ -0,0 +1,54 @@
+// const { flattenJson } = require("../../v0/util");
+const { Utils } = require("rudder-transformer-cdk");
+
+function commonPostMapper(event, mappedPayload, rudderContext) {
+  const { message, destination } = event;
+  const payload = mappedPayload;
+
+  // If user provided a eventType name, then we will include it in the payload directly
+  if (destination.Config.customEventType) {
```

Comment:
Apart from using "destination.Config.xxx" store Config in some variable and use that.

Reply:
Done